### DISCUSSION TITLE
thanhle.blog/blockchain-concept-cho-web2-token-erc20

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#74] [Ideas]
# thanhle.blog/blockchain-concept-cho-web2-token-erc20

Hê sờ lô, hê sờ ly ly các bạn, tuần này tìm hiểu xíu về cách đống token hoạt động như nhế nào trên blockchain nhé

https://thanhle.blog/blog/blockchain-concept-cho-web2-token-erc20

**Comment by thanhlmm** [comment]
Ừm bài viết vẫn kiểu hơi khó hỉu, nhưng chưa biết sao để improve 😅
